lib/index.js

```diff
@@ -149,16 +143,16 @@ export default {
 
         if (foundConfig) {
           // We have a configuration from Stylelint
-          options.config = assignDeep(rules, foundConfig.config);
+          options.config = foundConfig.config;
```

Comment:
Note that in addition to simplifying this code a bit, this _might_ have fixed a potential bug when a user **did** provide a configuration and it used `extends`.